docs/src/content/pages/how-keystatic-organises-your-content.mdoc

```diff
@@ -35,33 +35,84 @@ export default config({
 
 The optional trailing slash `/` on that path has an impact on the content structure - read below for more details on `collection paths` and `singleton paths`.
 
-## Collections
+### Path wildcard for collections
 
-The `path` property for collections is a string that contains a `*` wildcard representing the `slug` of an entry.
+The `path` property for `collections` must contain a `*` wildcard. 
 
-If not specified, the default `path` value will be `{collection-name}/*/`.
+It represents the `slug` of an entry.
 
-{% aside icon="⚡️" %}
-**Why a** `*` **symbol?**
+This wildcard gives you flexibility and control over your where your content is being output.
+
+### Nested folder output example
+
+{% aside icon="👉" %}
+  `path: 'packages/design-system/*/docs/'`
+{% /aside %}
+
+Imagine a Design System inside a monorepo: 
+
+```sh
+root
+├── packages
+  ├── design-system
+    ├── button
+      └── src
+    ├── dropdown
+      └── src
+└── apps
+  └── docs(keystatic)
+```
 
-The `*` wildcard in the `path` is useful when you want to collocate Keystatic's output with your existing source code, that may be nested a few levels deep inside an ***entry*** you're writing content for.
+Your Keystatic site lives in `apps/docs`, but you want your documentation entries in `packages/design-system/{component-name}/docs/`, to collocate them with each component.
 
-For example, you may be writing documentation for Design System components.
+The following path will let you do exactly that:
 
-Say you want Keystatic to output your documentation entries in `packages/design-system/{component-name}/docs/`.
+```
+path: 'packages/design-system/*/docs/'
+```
 
-You can use `path: 'packages/design-system/*/docs/'` to do just that 👍
+### Nested slug example
+
+{% aside icon="👉" %}
+  `path: 'content/posts/**/*'`
 {% /aside %}
 
+There may be situations where you need the `slug` of an entry to be following a multi-folder structure.
+
+Say you want the **same collection** to support this following tree structure:
+
+```sh
+content
+├── posts
+  ├── en  
+      └── post-1.mdoc
+  ├── fr  
+      └── post-1.mdoc
+```
+
+You can enable this by using the `**/*` wildcard in your `path`: 
+
+```
+path: 'content/posts/**/*'
```

Comment:
````suggestion
You can enable this by using the `**` wildcard in your `path`: 

```
path: 'content/posts/**'
````

Reply:
Updated these 2 + another instance of `**/*` 👍